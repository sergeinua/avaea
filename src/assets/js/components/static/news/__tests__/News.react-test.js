import React from 'react';
import StaticNews from '../News';
import renderer from 'react-test-renderer';


test('StaticNews component test', () => {
  var mockUser = {email: "test@gmail.com", id: 58};
  var component = renderer.create(
    <StaticNews user={mockUser} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});
