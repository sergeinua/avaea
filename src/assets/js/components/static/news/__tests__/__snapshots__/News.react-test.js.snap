exports[`test StaticNews component test 1`] = `
<div>
  <div
    className="content static news">
    <h1
      className="skeleton">
      News Content
    </h1>
  </div>
</div>
`;
